sharepoint/sharepoint-ps/sharepoint-online/Get-SPOTenantRestrictedSearchAllowedList.md

```diff
@@ -0,0 +1,47 @@
+---
+external help file: sharepointonline.xml
+Module Name: Microsoft.Online.SharePoint.PowerShell
+online version: https://learn.microsoft.com/powershell/module/sharepoint-online/get-spotenantrestrictedsearchallowedlist
+applicable: SharePoint Online
+title: Get-SPOTenantRestrictedSearchAllowedList
+schema: 2.0.0
+author: praporwal
+ms.author: praporwal
+ms.reviewer:
+---
+
+# Get-SPOTenantRestrictedSearchAllowedList
+
+## SYNOPSIS
+
+The Get-SPOTenantRestrictedSearchAllowedList cmdlet will return all the sites that are in the allowed list in the tenant.
+
+
+## SYNTAX
+
+```powershell
+Get-SPOTenantRestrictedSearchAllowedList
+```
+
+## DESCRIPTION
+
+Restricted SharePoint Search gives an ability to Global/Tenant and SharePoint admins to Global/Tenant and SharePoint admins to enable/disable organization wide search. This control, when enabled offers up to 100 sites to be allowed in organization wide search, includes user’s previously accessed files and includes content from user’s frequent sites. Allow list is a set of curated sites where the customer has reviewed the permissions and has applied data governance on them. Allow list will support Site Collections, Hub and Comm sites. 
+
+The Get-SPOTenantRestrictedSearchAllowedList cmdlet will return all the sites that are in the allowed list in the tenant.  
+
+You must be a SharePoint Online or global administrator to run the Get-SPOTenantRestrictedSearchAllowedList cmdlet. 
```

Comment:
Does this line applies to all the commands?
Could we have this line everywhere? Seems important.

Reply:
Updated as per documentation shared by PM. Can check if they want to include at other places.

Reply:
@praporwal-microsoft these are some good PRs to reference -- these were previously reviewed by Vendors.

https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/208/files
https://github.com/MicrosoftDocs/OfficeDocs-SharePoint-PowerShell/pull/254/files